src/ocean/util/log/Config.d

```diff
@@ -315,7 +315,9 @@ public Layout newLayout ( istring layout_str )
             break;
 
         default:
-            throw new Exception("Invalid layout requested : " ~ layout_str);
+            istring msg = "Invalid layout requested : ";
```

Comment:
Why this code formatting change? Adding file and line is unrelated to this commit, right?